trin-core/src/portalnet/overlay_service.rs

```diff
@@ -1272,6 +1361,33 @@ where
         }
     }
 
+    /// Provide the requested content key and content value for the acceptor
+    fn provide_requested_content(
+        &self,
+        response: &Accept,
+        content_keys_offered: Vec<TContentKey>,
+    ) -> Vec<ByteList> {
+        let mut content_items: Vec<ByteList> = Vec::new();
+
+        for (i, key) in response
```

Comment:
The name `response` here is confusing to me to read in this context. Naturally, it makes sense in the context of an `accept` responding to an `offer`. But here, it seems like the response ought to be the thing we are building to respond to the `accept`. Maybe we could call this variable `accept` or `accept_message`?